### PR TITLE
console reporter added

### PR DESCRIPTION
**Pull request by StevenMaMei**
Third point of the workshop
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,11 +24,13 @@
...
   "homepage": "https://github.com/StevenMaMei/protractor-workshop-2020#readme",
-
   "devDependencies": {
     "protractor": "^7.0.0",
     "@types/jasminewd2": "^2.0.8",
     "@types/node": "^14.0.25",
     "typescript": "^3.9.7"
+  },
+  "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2"
```
Comment (holgiosalos) [inline package.json:32]: this dependency should be installed as dev dependency.

File: protractor/helpers/reporter.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { SpecReporter, StacktraceOption } from 'jasmine-spec-reporter';
+export let reporter = () => {
```
Comment (holgiosalos) [inline protractor/helpers/reporter.ts:3]: leave a blank line between import statements and the reporter definition.

File: protractor/local.config.ts [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { Config,browser } from 'protractor';
-
+import { reporter } from './helpers/reporter';
 export const config: Config = {
```
Comment (holgiosalos) [inline protractor/local.config.ts:4]: leave a blank line between the import statements and the config definition

File: protractor/local.config.ts [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
   seleniumAddress: 'http://localhost:4444/wd/hub',
   onPrepare: () => {
+    reporter();
```
Comment (holgiosalos) [inline protractor/local.config.ts]: this should go outside of the onPrepare function

File: package.json (hunk 1 above)
Comment (StevenMaMei) [reply package.json:32]: Done

File: protractor/helpers/reporter.ts (hunk 2 above)
Comment (StevenMaMei) [reply protractor/helpers/reporter.ts:3]: Done

File: protractor/local.config.ts (hunk 3 above)
Comment (StevenMaMei) [reply protractor/local.config.ts:4]: Done

File: protractor/local.config.ts (hunk 4 above)
Comment (StevenMaMei) [reply protractor/local.config.ts:8]: image

Done

Comment (StevenMaMei) [reply protractor/local.config.ts]: The line "reporter()" if I put outside of the onPrepare function, it throws an error.  The guide says to put it into the onPrepare function
image

Comment (holgiosalos) [reply protractor/local.config.ts]: You're right, I'm sorry, got confused.

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**